src/Fetcher/TululaFetcher.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Fetcher;
+
+use App\Entity\Conference;
+use App\Entity\Continent;
+use App\Entity\Tag;
+use App\Repository\ExcludedTagRepository;
+use App\Repository\TagRepository;
+use Psr\Log\LoggerInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class TululaFetcher implements FetcherInterface
+{
+    private const SOURCE = 'tulula';
+    private const TAG_ALLOW_LIST = [
+        'android',
+        'css',
+        'data',
+        'devops',
+        'dotnet',
+        'facebook',
+        'golang',
+        'html',
+        'ios',
+        'java',
+        'javascript',
+        'nodejs',
+        'php',
+        'python',
+        'react native',
+        'ruby',
+        'rust',
+        'security',
+        'ux',
+        'ai/ml',
+        'sql',
+        'iot',
+    ];
+
+    private LoggerInterface $logger;
+    private TagRepository $tagRepository;
+    private ExcludedTagRepository $excludedTagsRepository;
+    private LocationGuesser $locationGuesser;
+    private HttpClientInterface $client;
+
+    public function __construct(LoggerInterface $logger, LocationGuesser $locationGuesser, HttpClientInterface $client, TagRepository $tagRepository, ExcludedTagRepository $excludedTagRepository)
+    {
+        $this->client = $client;
+        $this->logger = $logger;
+        $this->tagRepository = $tagRepository;
+        $this->locationGuesser = $locationGuesser;
+        $this->excludedTagsRepository = $excludedTagRepository;
+    }
+
+    public function isActive(): bool
+    {
+        return true;
+    }
+
+    public function getUrl(array $params = []): string
+    {
+        return 'https://tulu.la/api/public';
+    }
+
+    public function fetch(): \Generator
+    {
+        $validConferences = [];
+        foreach ($this->queryTululaEvents() as $conference) {
+            foreach ($conference['tags'] as $tag) {
```

Comment:
I don't like the naming here : tag vs lowerTag vs $repositoryTag
Let use:
* $tagName instead of tag & lowerTag
* $tag instead of repositoryTag
I always use this "pattern", it works very well